cmd/configure.ac

```diff
@@ -257,5 +257,22 @@ fi
 fi
 AM_CONDITIONAL([USE_INTERNAL_BPF_HEADERS], [test "x$host_bpf_headers_usable" = "xno"])
 
+AC_DEFINE([MISSING_FIELD_INITIALIZERS_USABLE], [1], [-Wmissing-field-initializers are usable])
```

Comment:
It is the first time I see a test that verifies a warning works. Was it really the goal? Or should we just check that the flag does not break compilation?

I propose something like that:
```
AX_APPEND_COMPILE_FLAGS([ dnl
  -Wall dnl
  -Wextra dnl
  -Wmissing-field-initializers dnl
  -Wmissing-prototypes dnl
  -Wstrict-prototypes dnl
  -Wno-unused-parameter dnl
  ], [CHECK_CFLAGS])
AC_SUBST([CHECK_CFLAGS])
```

Reply:
Ah nevermind. The flag is broken.